<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>676</width>
    <height>498</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>676</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DAM UF1 Pe1 (Patricio Vidal)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="brmenys">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brmes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bpmenys">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bpmes">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBR">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>350</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBP">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="capcelera">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>591</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167);
color: white;</string>
    </property>
    <property name="text">
     <string>Ley de la palanca - Arqu√≠medes</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="palanca">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>91</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;palanca.jpg&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imatges_palanca/palanca.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>91</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdBR">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdBP">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="R">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>60</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>60</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imatges_palanca/arquimedes_palanca_mundo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderBR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdBR</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdBP</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
